### PR TITLE
Rahul Docx update

### PR DESCRIPTION
**Pull request by rahulmahendru**

### CODE REVIEW

File: src/pages/docs.jsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ import {
 import { useAuthState } from 'react-firebase-hooks/auth';
 import { join } from 'path';
 
+import * as fs from "fs";
```
Comment (nahtnam) [inline src/pages/docs.jsx]: remove this

File: src/pages/docs.jsx [hunk 2]
```diff
@@ -121,8 +123,30 @@ function App(props) {
     const name = getName(id) || id;
     const path = join(homedir, `${name}.txt`);
     fs.writeFileSync(path, getDocument(id));
-  };
+}
```
Comment (nahtnam) [inline src/pages/docs.jsx]: missing semicolon

**Review by nahtnam — CHANGES_REQUESTED** [review]
Tests are failing, fix that. Other than that, its good

**Comment by rahulmahendru** [comment]
Changes made

**Comment by nahtnam** [comment]
Still failing, run `npm run lint -- --fix` and commit that

**Comment by rahulmahendru** [comment]
Done!

**Comment by rahulmahendru** [comment]
Checks passed!

**Review by nahtnam — APPROVED** [review]

**Pull request merged**